turbinia/evidence.py

```diff
@@ -97,7 +96,12 @@ class Evidence(object):
         to the saved_path location.
     source (str): String indicating where evidence came from (including tool
         version that created it, if appropriate).
-    local_path (str): A string of the local_path to the evidence.
+    local_path (str): Path to the processed data (can be a blockdevice or a
+        directory).
+    device_path (str): Path to a relevant 'raw' data source (ie: a block
+        device or a raw disk image).
+    source_path (str): Path to the un-processed source data for the Evidence.
+    mount_path (str): Path to a mounted file system (if relevant).
```

Comment:
Why don't we want `mount_path` and `device_path` to be set in `RawDisk`?  These aren't relevant for a lot of the other evidence types.

Reply:
I agree for device_path.
mount_path points to a filesystem, so that could be nice for Directory & the likes. WDYT?

Reply:
Sure, SGTM

Reply:
Done